### ISSUE TITLE
Gradle build failed

### ISSUE THREAD

**Issue by PixelEnvision** [jizc/CloudOnce#16]
Hello again,

When I try to build with grade, it fails with the following error:

`/Users/test/Documents/Unity/kubic/Temp/gradleOut/MainLibProj/build/intermediates/bundles/release/aapt/AndroidManifest.xml:31:28-65: AAPT: No resource found that matches the given name (at 'value' with value '@integer/google_play_services_version').`

I think it cannot find a value to use for @integer/google_play_services_version in AndroidManifest.xml located in MainLibProj

Any suggestions to fix that?

Best,
E.
--
### Environment

- CloudOnce __v2.3.6__
- Unity3D __v5.5.3p4__
- Build platform: Android

**Comment by jizc** [comment]
This appears to be an issue with the Google Play Games plugin that is included in CloudOnce. I tried building an APK using just the [GPG plugin](https://github.com/playgameservices/play-games-plugin-for-unity) and I got the same error.

So I would suggest you make an issue over on [their repo](https://github.com/playgameservices/play-games-plugin-for-unity/issues). I will be happy to update CloudOnce once they add support for the Gradle build system.
